tests/python/unittest/test_meta_schedule_post_order_apply.py

```diff
@@ -371,8 +371,8 @@ def test_meta_schedule_custom_search_space():
     )
     post_order_apply = PostOrderApply()
     post_order_apply.initialize_with_tune_context(context)
-    with pytest.raises(ValueError, match="Custom schedule rule not found"):
-        post_order_apply.generate_design_space(mod)
+
+    post_order_apply.generate_design_space(mod)
```

Comment:
Would you please keep another test for missing custom schedule rule scenario?

Reply:
L375 is the missing custom schedule rule scenario. Previously it would raise error (caught by `with pytest.raises`).

Reply:
I see, so it will raise a warning first and then do the acutal work later in line 386. That make sense.